en/developer/customization/articles/50-liferay-forms/02-form-storage-adapters.markdown

```diff
@@ -117,34 +118,34 @@ Call the request object's getters to access the necessary data:
 
 - Method: `public String getUuid()`
     Returns the unique ID of the Form Record, which is useful for identifying
-    the form record across all other form records in the database. The first time
-    a record is saved, this returns `null`.
+    the form record across portal environments (like staged and live). This is
+    often needed to pass to the `ServiceContext` object used in a service call.
+    Before the record is persisted, this returns `null`.
 
 - Method: `public boolean isInsert()`
     Returns _true_ if the primary key is `0`, and _false_ if there's an existing
     ID for the form record. The default JSON storage adapter uses it to test
     whether an add or update is called for.
 
-`DDMStorageAdapterGetRequest`:
+`DDMStorageAdapterDeleteRequest`:
```

Comment:
@rbohl, why did you replace it by the `DDMStorageAdapterDeleteRequest`? This change will provide an incorrect info to the user once the `DDMStorageAdapterDeleteRequest` doesn't have the `getDDMForm` method, actually, this method if from the class `DDMStorageAdapterGetRequest`.

Reply:
Thanks, it looks like I had it right and then confused myself. Glad you caught it.